kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml

```diff
@@ -14,6 +14,7 @@
 	      	<status>Initiated</status>
 	      	<status>In Process</status>
 	      	<status>Canceled In Process</status>
+	      	<status>Canceled - In Process</status>
```

Comment:
Due to complications caused by the Database Refresh, further work on this is blocked for the time being